smarts/core/mission_planner.py

```diff
@@ -202,44 +202,48 @@ def _edge_ids(self, pose: Pose, lane_id: str = None):
         return edge_ids
 
     def cut_in_waypoints(self, sim, pose: Pose, vehicle):
-        radius = self._mission.task.trigger_radius
+
+        aggressiveness = 5
```

Comment:
Need to rebase master and use the value passed in mission planner

Reply:
Thanks, sure.